pika/connection.py

```diff
@@ -26,6 +27,7 @@
 from pika import spec
 
 from pika.compat import xrange, basestring, url_unquote, dictkeys
+from pika.compat import dict_iteritems
```

Comment:
At this point, why not from ``pika import compat``? I don't consider this a blocker, just more of a thought about the coding style of importing a bunch of methods like that.